repository/src/main/java/tech/ydb/yoj/repository/db/EntityDescriptor.java

```diff
@@ -0,0 +1,15 @@
+package tech.ydb.yoj.repository.db;
+
+import tech.ydb.yoj.databind.schema.Schema;
+
+import javax.annotation.Nullable;
+
+public record EntityDescriptor<E extends Entity<E>>(Class<E> clazz, @Nullable String tableName) {
+    public String getTableName(Schema<?> schema) {
```

Comment:
- Why not `EntitySchema<?> schema`? Nobody should call **Entity**Descriptor to determine table name for something which is not an entity!
- Remember to add @NonNull lombok annotation to `schema` parameter